marketplaceItems/voiceScopeApp/README.md

```diff
@@ -0,0 +1,14 @@
+# The Voice Scope App
+
+Have you ever had trouble figuring out who was talking in a group? Then this is the tool for you! The Voice Scope is a tool that lets you see the audio output of the people around you visualized as a loudness meter floating above their head.
+
+Simply enable the app by clicking the ON/OFF button and enjoy the knowledge of who is talking, visualized.
+
+# TO DO
+
+1. Allow app to persist on domain changes.
+2. Clean up userArray and userObject if sorting the loudest users is not desired.
+3. Remove the "findOverlays()" usage for cleaning up straggler overlays once certain deleting parents takes care of children.
+4. Remove sorting function if not sorting anything.
+5. Figure out creation of overlays with children and why there is a non-zero amount of time that needs to pass before spawning children overlays.
+
```

Comment:
Please add a release notes section as per [this link](https://highfidelity.atlassian.net/wiki/spaces/CPT/pages/441974999/Our+hifi-content+Repo+Release+Procedure#Ourhifi-contentRepo&ReleaseProcedure-ReleasingMarketplaceItems).

Note that this means you'll need two commits (unless you amend the commit): one to fix the issues in this review, and one to add the commit SHA associated with your release of the code after fixing the issues.

if you need clarification, please ask!